src/utils/clmfates_interfaceMod.F90

```diff
@@ -583,8 +596,12 @@ subroutine dynamics_driv(this, nc, bounds_clump,      &
       type(soilbiogeochem_carbonflux_type), intent(inout) :: soilbiogeochem_carbonflux_inst
       type(frictionvel_type)  , intent(inout)        :: frictionvel_inst
 
+      type(fates_fire_data_type) :: fates_fire_data_inst
+      type(fates_fire_no_data_type) :: fates_fire_no_data_inst
```

Comment:
Here (and possibly elsewhere) I entered `fates_fire_data_inst` and `fates_fire_no_data_inst` via `use` statements (see a few lines up) rather than as arguments through the call. Will this cause trouble?

Reply:
You actually don't want these at all. You will reference them through the generic type of fates_fire_data_method, so you don't have to know which specific type it is. Nor do you have to declare each one.

Reply:
Same [comment](https://github.com/ESCOMP/CTSM/pull/991#discussion_r415423650)